docstring_parser/parser.py

```diff
@@ -19,16 +24,39 @@ def parse(text: str, style: DocstringStyle = DocstringStyle.auto) -> Docstring:
     :returns: parsed docstring representation
     """
     if style != DocstringStyle.auto:
-        return STYLES[style](text)
+        return STYLES[style].parse(text)
 
     rets = []
-    for parse_ in STYLES.values():
+    for module in STYLES.values():
         try:
-            rets.append(parse_(text))
+            rets.append(module.parse(text))
         except ParseError as e:
             exc = e
 
     if not rets:
         raise exc
 
     return sorted(rets, key=lambda d: len(d.meta), reverse=True)[0]
+
+
+def compose(
+    docstring: Docstring,
+    style: DocstringStyle = DocstringStyle.auto,
+    rendering_style: RenderingStyle = RenderingStyle.compact,
+    indent: str = "    ",
+) -> str:
+    """Render a parsed docstring into docstring text.
+
+    :param docstring: parsed docstring representation
+    :param style: docstring style to render
+    :param indent: the characters used as indentation in the docstring string
+    :returns: docstring text
+    """
+    if style != DocstringStyle.auto:
+        return STYLES[style].compose(
+            docstring, rendering_style=rendering_style, indent=indent
+        )
+    else:
```

Comment:
Unnecessary `else:` after `return`

In fact I'd assign it like so:

```python3
module = _STYLE_MAP[docstring.style if style == Docstring.auto else style]
return module.compose(…)
```

Reply:
ok.